src/backend/InvenTree/order/models.py

```diff
@@ -523,6 +523,37 @@ def company(self):
         help_text=_('Date order was completed'),
     )
 
+    approved_by = models.ForeignKey(
+        User,
+        on_delete=models.SET_NULL,
+        blank=True,
+        null=True,
+        verbose_name=_('Approved by'),
+    )
+
+    approved_date = models.DateField(
+        blank=True,
+        null=True,
+        verbose_name=_('Approve Date'),
+        help_text=_('Date order was approved'),
+    )
+
+    reject_reason = models.CharField(
+        max_length=128,
+        blank=True,
+        verbose_name=_('Reason for rejection'),
+        help_text=_('The reason for rejecting this order'),
+    )
+
+    placed_by = models.ForeignKey(
+        User,
+        related_name='orders_placed',
```

Comment:
This field is only present in purchase orders, the related name could be confusing to plugin creators

Reply:
Fair point. I'll rename it.